single-node.sh

```diff
@@ -142,10 +142,9 @@ build)
         info "cloning services git repos"
         clone_git_repos
 
-        info "cloning debootstap git repo"
+        info "setting up toolset"
         pushd "${TARGET}"/pieman
-            sudo -u "${USER}" git clone https://anonscm.debian.org/git/d-i/debootstrap.git
-            sudo -u "${USER}" git -C debootstrap checkout "${DEBOOTSTRAP_VER}"
```

Comment:
I wonder if we use `DEBOOTSTRAP_VER` somewhere else. Can you please check where its defined and remove if not used?